### PR TITLE
bringing group mentoring program current

### PR DESCRIPTION
**Pull request by parispittman**
Co-authored-by: Elana Hashman <ehashman@redhat.com>

### CODE REVIEW

File: mentoring/programs/group-mentoring.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-
-## Summary
-This is an experimental group mentoring program where individuals will work
-self-paced in a semi-structured learning environment over the course of three
+# Contributor Ladder Growth Programs
+  
+These programs place an emphasis on learning in groups for active contributors
+that would like to be a reviewer or other leadership roles like a Chair. The 
+groups are semi-structured learning environments over the course of three
 months. Want to get `/lgtm` rights? Be a decision maker in an OWNERS file? This
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:6]:
```suggestion
groups are semi-structured learning environments over the course of a
release. Want to get `/lgtm` rights? Be a decision maker in an OWNERS file? This
```

Suggest we lock this to the release cycle.

File: mentoring/programs/group-mentoring.md [hunk 2]
```diff
@@ -13,29 +11,25 @@ Kubernetes contributors.
...
 ## How This Works
 Foundation:  
-This program is built around our [community membership guidelines].
-This doc shows how you can grow through the project in different roles and
+This program is built around our [community membership guidelines] as well as 
+our [Chair] and [Tech Lead] governance descriptions.
+These docs show how you can grow through the project in different roles and
 provides the requirements to get to each level on our ladder. Building trust is
-the key to this document.
+the key.
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:18]:
```suggestion
key.
```

File: mentoring/programs/group-mentoring.md [hunk 3]
```diff
@@ -13,29 +11,25 @@ Kubernetes contributors.
...
-TODO: make the self service aspect here solid for faster onboarding
-
-Mentors can rotate facilitating once a week for slack standups with the expectation
-that cohort members need to check in with a predetermined status update
-(accomplishments, challenges, etc)  
+- No more than 4 mentees per 1 mentor; no more than 8 total in the group
+  - this scales better than 1:1 and peers can help each other in a community
+  setting
+- Everyone is on the same journey 
+Member -> Reviewer | Reviewer -> Owner | Member/Reviewer/etc -> Chair
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:25]:
```suggestion
Member -> Reviewer | Reviewer -> Approver | SIG Member -> Chair
```

File: mentoring/programs/group-mentoring.md [hunk 4]
```diff
@@ -13,29 +11,25 @@ Kubernetes contributors.
...
+- Three months
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:26]:
```suggestion
- Three months/one release cycle
```

File: mentoring/programs/group-mentoring.md [hunk 5]
```diff
@@ -13,29 +11,25 @@ Kubernetes contributors.
...
+- Private slack channel
+ Folks want to know you are dependable and understand the domain.
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:28]:
```suggestion
 For you to progress, the community wants to know you are dependable and understand the domain.
```

File: mentoring/programs/group-mentoring.md [hunk 6]
```diff
@@ -47,38 +41,38 @@ that cohort members need to check in with a predetermined status update
  
 
 ## Mentor requirements  
-TODO  
+- Github Org Member
+- At least the same level as the goal of the group (example: Reviewers for
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:45]:
```suggestion
- At least the same level as the goal of the group (example: Reviewer for 
```

File: mentoring/programs/group-mentoring.md [hunk 7]
```diff
@@ -47,38 +41,38 @@ that cohort members need to check in with a predetermined status update
  
 
 ## Mentor requirements  
-TODO  
+- Github Org Member
+- At least the same level as the goal of the group (example: Reviewers for 
+Reviewers) 
+- Time committment: one dedicated day of slack triage and at least a 30 minute
```
Comment (ehashman) [inline mentoring/programs/group-mentoring.md:47]:
```suggestion
- Time commitment: facilitate biweekly, hour long zoom sessions plus curriculum planning in between sessions (~1-2h/wk total).
```

**Review by ehashman — CHANGES_REQUESTED** [review]

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Review by ehashman — COMMENTED** [review]
/lgtm

**Pull request merged**